src/sass/shared/variables.scss

```diff
@@ -85,11 +91,21 @@ $font-size-title-xsmall: 21px;
 $font-size-small: 18px;
 $font-size-xsmall: 16px;
 $font-size-xxsmall: 14px;
+$font-size-micro: 12px;
 $font-size-footer-titles: 21px;
 $font-size-footer-text: $font-size-xsmall;
 $font-size-secondary-nav-item: 21px;
 $font-size-testimonial-usernames: 18px;
 
+// Line Height
+$line-height-condensed: 1em;
+$line-height-small: 1.2em;
+$line-height-projects-title: 30px;
+
+// Font weight
+$font-weight-bold: bold;
```

Comment:
Font weights are not gonna change. Either they're bold or they're not. I don't see the point of having variables for this